src/Handler/Table/Create/CreateTableHandler.php

```diff
@@ -63,25 +66,70 @@ public function __invoke(
                 'nullable' => $column->getNullable(),
                 'default' => $column->getDefault() === '' ? null : $column->getDefault(),
             ]);
-            $columns[] = new BigqueryColumn($column->getName(), $columnDefinition);
+            $createTableOptions['schema']['fields'][] = SQLtoRestDatatypeConverter::convertColumnToRestFormat(
+                new BigqueryColumn($column->getName(), $columnDefinition)
+            );
+        }
+
+        $meta = MetaHelper::getMetaFromCommand(
+            $command,
+            CreateTableCommand\BigQueryTableMeta::class
+        );
+        if ($meta !== null) {
+            assert($meta instanceof CreateTableCommand\BigQueryTableMeta);
+            if ($meta->getTimePartitioning() !== null) {
+                $timePartitioningOptions = [
+                    'type' => $meta->getTimePartitioning()->getType(),
+                ];
+                if ($meta->getTimePartitioning()->getExpirationMs() !== null) {
+                    $timePartitioningOptions['expirationMs'] = $meta->getTimePartitioning()->getExpirationMs();
+                }
+                if ($meta->getTimePartitioning()->getField() !== null) {
+                    $timePartitioningOptions['field'] = $meta->getTimePartitioning()->getField();
+                }
+                $createTableOptions['timePartitioning'] = $timePartitioningOptions;
+            }
+            if ($meta->getClustering() !== null) {
+                $createTableOptions['clustering'] = [
+                    'fields' => ProtobufHelper::repeatedStringToArray($meta->getClustering()->getFields()),
+                ];
+            }
+            if ($meta->getRangePartitioning() !== null) {
+                assert($meta->getRangePartitioning()->getRange() !== null);
+                $createTableOptions['rangePartitioning'] = [
+                    'field' => $meta->getRangePartitioning()->getField(),
+                    'range' => [
+                        'start' => $meta->getRangePartitioning()->getRange()->getStart(),
+                        'end' => $meta->getRangePartitioning()->getRange()->getEnd(),
+                        'interval' => $meta->getRangePartitioning()->getRange()->getInterval(),
+                    ],
+                ];
+            }
```

Comment:
Toto by slo vycuknut niekam do triedy a napisat napriklad na to unit test.

Reply:
gut idea, vyhodil sem ven a přidal test